code/modules/projectiles/guns/launcher/bows.dm

```diff
@@ -0,0 +1,156 @@
+/obj/item/weapon/arrow/wood
+	name = "wooden arrow"
+	desc = "A wooden arrow with a stone tip, simple, but gets the job done"
```

Comment:
```suggestion
	desc = "A wooden arrow with a stone tip, simple, but gets the job done."
```
(Please check punctuation elsewhere, this is just the first that caught my eye)

Reply:
Would also argue changing the first "," to a period as well. so "A wooden arrow with a stone tip. Simple, but gets the job done."